source/content/advanced-redirects.md

```diff
@@ -0,0 +1,334 @@
+---
+title: Advanced Redirects and Restrictions
+description: Configure custom redirect logic for specific scenarios
+tags: [redirects, variables, dns]
+categories: [go-live,develop]
+reviewed: "2020-02-12"
+---
+
+Basic domain and HTTPS redirection can be handled by the [Primary Domain]() feature. PHP redicts can be used if these configurations are not an option, or when you need specific redirect logic. Depending on the needs of the site, you may only need one, some, or none of the following.
+
+Redirect logic should be added to `wp-config.php` for WordPress sites, and `settings.php` for Drupal sites. Note that the platform-set primary domain will redirect all requests, not just the root domain.
```

Comment:
Consider using a warning or note for the second sentence here - worth a stronger call out imo

Reply:
Noted: https://github.com/pantheon-systems/documentation/pull/5516/commits/a7e4c20988a819b75567c926d7678d71063e686a